plugins/importexport/native/filter/NativeImportFilter.inc.php

```diff
@@ -98,6 +89,21 @@ function handleElement($node) {
 	function parseLocalizedContent($element) {
 		return array($element->getAttribute('locale'), $element->textContent);
 	}
+
+	function importWithXMLNode($n, $filter = null) {
+		$articleGalleyDoc = new DOMDocument();
```

Comment:
I don't think `$articleGalleyDoc` is appropriate here.

Reply:
Fixed, and added function signature